Core/Kernel/elxElastixMain.cxx

```diff
@@ -811,9 +811,8 @@ ElastixMain::CreateComponent(
 {
   /** A pointer to the New() function. */
   PtrToCreator  testcreator = 0;
-  ObjectPointer testpointer = 0;
   testcreator = this->s_CDB->GetCreator( name,  this->m_DBIndex );
-  testpointer = testcreator ? testcreator() : NULL;
+  ObjectPointer testpointer = testcreator ? testcreator() : ObjectPointer();
```

Comment:
Just to explain: `ObjectPointer()` is a default-initialized smart pointer, equivalent to NULL.

ITK5 SmartPointer no longer supports initialization by `NULL`.